src/java/picard/analysis/artifacts/CollectSequencingArtifactMetrics.java

```diff
@@ -138,7 +138,7 @@ public static void main(final String[] args) {
     }
 
     @Override
-    protected void setup(final SAMFileHeader header, final File samFile) {
+    protected void setup(final SAMFileHeader header, final File samFile, final File referenceSequence) {
```

Comment:
Note that this change also breaks the compile for personal tools in picard private (which are not built by the ant scripts).  Not sure what protocol is for that.